issue 7 & 3 - Fixed python version, cqlsh, and healthcheck

dstoft opened the pull request:
Fixed:
* python version, it now uses python 2.7, which is the only version supported by Cassandra. It installs pip using the newly added get-pip.py. Dockerfile has been updated accordingly.
* configure-cassandra.yml updated to python 2.7 syntax
* CQLSH logs in using the default user and password, which is never changed atm. Therefore, exposing them here is not critical. This can be circumvented using cassandra.yml, environment variables and random generated password on startup. Visible in healthcheck.sh.
* Healthcheck fixed, and works now.
* start.sh listens to sigterm and stops cassandra gracefully.

dstoft committed:
Fixed python version, cqlsh, and healthcheck

Fixed:
* python version, it now uses python 2.7, which is the only version supported by Cassandra. It installs pip using the newly added get-pip.py. Dockerfile has been updated accordingly.
* configure-cassandra.yml updated to python 2.7 syntax
* CQLSH logs in using the default user and password, which is never changed atm. Therefore, exposing them here is not critical. This can be circumvented using cassandra.yml, environment variables and random generated password on startup. Visible in healthcheck.sh.
* Healthcheck fixed, and works now.
* start.sh listens to sigterm and stops cassandra gracefully.

mads5606 reviewed (commented):
Looks really good. I only have one question for the start.sh file: If stopdaemon fails do we get the error-message from it properly as exit status? Im honestly not sure how that works with ??

dstoft commented:
I honestly don't know either. Do you have any suggestions with your knowledge on the signal handling from the other containers? Is there a way that you can change this, such that we know for sure that we will get a error message?

mads5606 commented:
> I honestly don't know either. Do you have any suggestions with your knowledge on the signal handling from the other containers? Is there a way that you can change this, such that we know for sure that we will get a error message?

I think wrapping it in a function and trapping with that might work, but not sure. Will try and test it later this week.

mads5606 committed:
Put trap in a function

Encapsulated the trap in a function to save
the exit result from stopdeamon, make sure we
get that as the exit status if it happens to fail.

mads5606 commented:
@dstoft I put the trap in a function, so should get the stopdeamon return value as exit status in all cases. If you agree with the changes I think we can just merge it all
